### PR TITLE
Add Article content type

### PR DESCRIPTION
**Pull request by kallehauge**
This PR is basically a huge stack of configuration-files doing the following:
- Update `composer.json` to include _Paragraphs_ and _Video_ modules.
- Enable the following modules:
  - Paragraphs (and entity_reference_revisions).
  - Field UI.
  - Taxonomy.
  - Image.
  - Video.
- Create a new content type called _Article_ with fields:
  - Title.
  - Article type (taxonomy ref).
  - Author (textfield).
  - Content (paragraphs ref).
- Create a taxonomy called _Article types_.
- Add Paragraphs types:
  - Text: contains a textarea field.
  - Image: contains an image field.
  - Embedded Video: contains a video field for embed url.

### CODE REVIEW

File: web/config/sync/core.base_field_override.node.article.promote.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+required: false
+translatable: true
+default_value:
+  -
+    value: 0
+default_value_callback: ''
+settings:
+  on_label: 'On'
+  off_label: 'Off'
+field_type: boolean
```
Comment (kasperg) [inline web/config/sync/core.base_field_override.node.article.promote.yml]: Can we remove the promote field? It does not seem to make much sense for us.

File: web/config/sync/core.entity_form_display.node.article.default.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      add_mode: button
+      form_display_mode: default
+    third_party_settings: {  }
+  promote:
+    type: boolean_checkbox
+    settings:
+      display_label: true
+    weight: 6
+    third_party_settings: {  }
+  sticky:
```
Comment (kasperg) [inline web/config/sync/core.entity_form_display.node.article.default.yml]: Can we remove the sticky and promote fields? They do not seem to make much sense for us.

File: web/config/sync/core.entity_form_display.paragraph.embedded_video.default.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+content:
+  field_embedded_video:
+    weight: 0
+    settings:
+      allowed_providers:
+        vimeo: vimeo
+        youtube: youtube
+        dailymotion: 0
+        facebook: 0
+        instagram: 0
```
Comment (kasperg) [inline web/config/sync/core.entity_form_display.paragraph.embedded_video.default.yml]: We could consider enabling facebook and perhaps instagram as well if they work.

File: web/config/sync/core.entity_form_display.paragraph.text.default.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+bundle: text
+mode: default
+content:
+  field_textarea:
+    weight: 0
+    settings:
+      rows: 8
+      placeholder: ''
+    third_party_settings: {  }
+    type: text_textarea
```
Comment (kasperg) [inline web/config/sync/core.entity_form_display.paragraph.text.default.yml]: How about using the (limited) WYSIWYG filter here?

**Comment by kasperg** [comment]
I think we should enable our (limited) WYSIWYG editor for the textarea fields.

**Comment by kasperg** [comment]
Hard to review but looks good.

I have a few questions and I think you should show it to @AndersFriisBrodsgaard as well to get his input.

File: web/config/sync/field_ui.settings.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+field_prefix: field_
+_core:
+  default_config_hash: Q1nMi90W6YQxKzZAgJQw7Ag9U4JrsEUwkomF0lhvbIM
```
Comment (kasperg) [inline web/config/sync/field_ui.settings.yml]: Why do we need the `field_ui` module?

**Comment by kallehauge** [comment]
I swapped the Video module with Embed Video Field since the first mentioned had a lot of cases where it would break and the code looked a bit poor, after a quick review.

I also made the limited WYSIWYIG editor to the default for textareas.

@kasperg what part of this review do you want me to run past Anders tomorrow? The fields or the option for Vimeo and Youtube? I'm not quite sure :)

**Comment by kasperg** [comment]
We will have to show this to him during demo. Reviewed and approved.

**Pull request merged**